.github/workflows/ci.yml

```diff
@@ -31,10 +31,12 @@ jobs:
           8
           17
           20
+          21
```

Comment:
Would you please split out the JavaSE-21ci change in to separate PR so we can land it immediately? Please remove Java 20 while on it too.

Reply:
Done in https://github.com/eclipse-jdt/eclipse.jdt.core/pull/1994